haiku-games/benettris/benettris-0.2.recipe

```diff
@@ -0,0 +1,46 @@
+SUMMARY="Tetris client and server"
+DESCRIPTION="BeNetTris is a multiplayer Tetris game that you can play \
+with your friends! BeNetTris is client and server compatible with V1.13 \
+or V1.14 of Tetrinet or Tetrinet TetriFast."
+HOMEPAGE="https://github.com/HaikuArchives/BeNetTris"
+COPYRIGHT="2000-2008 Cedric Vincent"
+LICENSE="GNU GPL v2"
+REVISION="1"
+srcGitRev="5d04121340b7c730160d5eab6a97c79b97168464"
+SOURCE_URI="https://github.com/HaikuArchives/BeNetTris/archive/$srcGitRev.tar.gz"
+CHECKSUM_SHA256="bd3096b84601a257cd22818ba0396be600d4b99e2f9d02f000d029502af9eddc"
+SOURCE_FILENAME="BeNetTris-$portVersion-$srcGitRev.tar.gz"
+SOURCE_DIR="BeNetTris-$srcGitRev"
+
+ARCHITECTURES="x86_gcc2 x86_64"
+
```

Comment:
Please add [`USER_SETTINGS_FILES`](https://github.com/haikuports/haikuports/wiki/HaikuPorter-BuildRecipes#user_settings_files) containing configuration files/folders that the program puts in the user home directory.